modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/resource/v1_0/TaxonomyCategoryResourceImpl.java

```diff
@@ -340,9 +341,9 @@ public TaxonomyCategory postTaxonomyCategoryTaxonomyCategory(
 			parentTaxonomyCategoryId);
 
 		return _addTaxonomyCategory(
-			assetCategory.getGroupId(), assetCategory.getDefaultLanguageId(),
-			taxonomyCategory, assetCategory.getCategoryId(),
-			assetCategory.getVocabularyId());
+			null, assetCategory.getGroupId(),
```

Comment:
We should allow specifying the externalReferenceCode from the old endpoints as well, so here we should get the taxonomyCategory.getExternalReferenceCode() instead of null

Reply:
🤦 That's right! I'm changing it. Thanks